packages/calendar-utils/src/calendar.ts

```diff
@@ -21,7 +20,7 @@ export const getPaddingDayCount = (day: MomentInput, locale: string) => {
   const paddingDayCount = (firstDayOfMonth - firstDayOfWeek + 7) % 7;
 
   // ensure number is always positive
```

Comment:
**nit**: now this message is a bit misleading

Reply:
Thank you for pointing it out, I think it's still true - we always return a number in range 0-6.

Reply:
Yes, you're right. 
What I was trying to highlight is that the comment, which refers to the next line, does not match the next instruction: we previously made sure that assumption with the call to `inRange(paddingDayCount, 0, 6)` but the the next call just makes sure `paddingDayCount` is a number, it does not validate what type of number is.

Anyway, this is just a tiny recommendation from my part.

Reply:
Ah, I understand now. Thanks for the explanation; it makes perfect sense to me!